extensions/industry/healthcare/fhir/datatypes/address.schema.json

```diff
@@ -0,0 +1,75 @@
+{
+   "meta:license":[
+      "Copyright 2024 Adobe Systems Incorporated. All rights reserved.",
+      "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+      "you may not use this file except in compliance with the License. You may obtain a copy",
+      "of the License at https://creativecommons.org/licenses/by/4.0/"
+   ],
+   "$schema":"http://json-schema.org/draft-06/schema#",
+   "$id":"https://ns.adobe.com/xdm/industry/healthcare/fhir/datatypes/address",
+   "title":"Address",
+   "type":"object",
+   "description":"An address expressed using postal conventions (as opposed to GPS or other location definition formats).(This datatype is created as per HL7 FHIR Release 5 specifications)",
+   "definitions":{
+      "address":{
+         "properties":{
+            "xdm:use":{
+               "title":"Use",
+               "type":"string",
```

Comment:
Since this is a "code" type in FHIR, the values are restricted only for certain allowed strings such as "home", "work", "temp" etc. So add an `enum` and `meta:enum` elements after "type":"string" as follows:

"enum": ["home", "work", "temp", "old", "billing],
 "meta:enum": {
    "work": "Work",
    "temp": "Temp",
    "old": "Old",
    "billing": "Billing"
}

Here the `enum` element is JSON Schema specific, whereas `meta:enum` element is XDM specific to help the XDM UI to render human readable equivalent of the enum value